### ISSUE TITLE
Plot the loss and accuracy given a neural net for our problem

### ISSUE THREAD

**Issue by ya332** [drexelai/binary-search-in-neural-nets#7]
Once a pipeline to save train and test results are done, we need to figure out a way to plot the results of a training/test phases.

**Comment by dsbuddy** [comment]
I can pick this one up

**Comment by ya332** [comment]
Invited to the repo with write access. Feel free to create a PR.
